src/agent/src/network.rs

```diff
@@ -3,8 +3,17 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+use anyhow::{anyhow, Context, Result};
+use nix::mount::{self, MntFlags, MsFlags};
 use protocols::types::{Interface, Route};
+use slog::Logger;
 use std::collections::HashMap;
+use std::fs;
+
+use crate::Sandbox;
+
+pub const KATA_GUEST_SANDBOX_DNS_FILE: &str = "/run/kata-containers/sandbox/resolv.conf";
```

Comment:
Does this need to be `pub`? It only appears to be used by this module?

Reply:
No, there is no `pub` need.